discovery-service/src/main/java/org/zowe/apiml/discovery/GatewayNotifier.java

```diff
@@ -185,6 +185,9 @@ public void run() {
                     notification.process();
                 }
             } catch (InterruptedException e) {
+                log.warn("Thread for Gateway notification was interrupted", e);
```

Comment:
We should use externalized messages for warn, info and error.

Reply:
done